eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,116 @@
+$Language = "go"
+$packagePattern = "go.mod"
+# rewrite from ChangeLog-Operations.ps1 used in Get-ChangeLogEntriesFromContent for go uses vx.x.x as version number
+$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+\s+(?<version>v$([AzureEngSemanticVersion]::SEMVER_REGEX))(\s+(?<releaseStatus>\(.+\))))"
+# constants for go version fetch
+$GO_VERSION_REGEX = ".+\s*=\s*`".*v?(?<version>$([AzureEngSemanticVersion]::SEMVER_REGEX))`""
+$VERSION_FILE_SUFFIXS = "*constants.go", "*version.go"
```

Comment:
Given we are only using this in the one place below I'd suggest moving it there and also I wouldn't capitalize the variable name in this case as that is more for constants.